app/src/instantsearch.js

```diff
@@ -332,5 +345,23 @@ class InstantSearch {
     removeCSS(this._temporaryHidingCSS);
     delete this._temporaryHidingCSS;
   }
+
+  _addClickListeners(clickAnalyticsEnabled) {
+    const articles = document.getElementById('algolia-hits');
+    for (let i = 0; i < articles.children[0].children.length; i++) {
+      const singleArticle = articles.children[0].children[i].firstElementChild;
+      const objectID = singleArticle.getAttribute('data-algolia-objectid');
+      const position = singleArticle.getAttribute('data-algolia-position');
+      const queryID = singleArticle.getAttribute('data-algolia-queryid');
+      //console.log(singleArticle, objectID, position, queryID);
+      singleArticle.addEventListener('click', function () {
+        if (clickAnalyticsEnabled) {
+          console.log('click', objectID, position, queryID);
+          trackClick(objectID, position, queryID);
```

Comment:
You could store the index name used in the constructor in an instance variable `this.indexName`, allowing you to pass it to `trackClick` easily.